pkg/sdn/plugin/networkpolicy.go

```diff
@@ -206,14 +199,28 @@ func (np *networkPolicyPlugin) syncNamespace(npns *npNamespace) {
 	otx := np.node.oc.NewTransaction()
 	otx.DeleteFlows("table=80, reg1=%d", npns.vnid)
 	if inUse {
-		if npns.isolated {
+		defaultAllow := true
```

Comment:
Can we get a comment on this block?  Your commit message has great information about how this works, so just plop it in here too?